atomix/cluster/src/main/java/io/atomix/raft/impl/RaftContext.java

```diff
@@ -125,7 +125,7 @@
   // Used for randomizing election timeout
   private final Random random;
   private PersistedSnapshot currentSnapshot;
-  private volatile HealthStatus health = HealthStatus.HEALTHY;
+  private volatile HealthReport health = HealthReport.healthy(this);
```

Comment:
:thought_balloon: You can probably suppress this warning via `@SuppressWarning` annotation. I don't remember why we didn't do it yet. Could you please try it?

Reply:
I think it wasn't a warning before because volatile enums are probably fine? I'll add a suppression here.